domain/src/main/assets/fractions_subtopic_0.json

```diff
@@ -0,0 +1,30 @@
+{
+  "subtopic_title": "What is Fraction?",
+  "next_subtopic_dict": {
```

Comment:
@BenHenning This is the major change in this new json parsing structure, but I amnot sure how much important this is for us.

Reply:
I'm not actually sure what this is for--are users expected to cycle through multiple revision cards at a time?

Reply:
After checking the web, https://www.oppia.org/subtopic/Multiplication/1, this content is used to show next card.

<img width="1680" alt="Screenshot 2020-07-15 at 2 20 56 AM" src="https://user-images.githubusercontent.com/9396084/87475159-e5df1e00-c641-11ea-8d5d-d224fc2d9997.png">